MMR.Randomizer/ConfigurationProcessor.cs

```diff
@@ -2,13 +2,19 @@
 using MMR.Randomizer.Models.Settings;
 using MMR.Randomizer.Utils;
 using System;
+using System.IO;
 
 namespace MMR.Randomizer
 {
     public static class ConfigurationProcessor
     {
         public static string Process(Configuration configuration, int seed, IProgressReporter progressReporter)
         {
+            if ( !Directory.Exists("Resources"))
```

Comment:
Maybe this should be `if (!Directory.Exists(Path.Combine(Values.MainDirectory, "Resources")))` instead?